gamedb.yaml

```diff
@@ -4104,6 +4119,10 @@
   platform: epic-store
   id: 7c566548d24343e48151125ced4b494c
   banner: steam:314660
+  background: https://cdn2.steamgriddb.com/hero/b257b3937f0e272207bf484c8e0beb59.jpg
```

Comment:
Why add background, icon, logo? They should be fetched through steam.

Reply:
It didn't found these for this game. 
Do you think this is a bug and I should rather open an issue?